mindmeld/cli.py

```diff
@@ -85,6 +98,133 @@ def _app_cli(ctx):
         ctx.obj = {}
 
 
+def _dvc_add_helper(filepath):
+    """
+    Returns True if successful, False otherwise along with helper message
+    :param path:
+    :return:
+    """
+    p = subprocess.Popen(
+        ["dvc", "add", filepath], stdout=subprocess.PIPE, stderr=subprocess.PIPE
+    )
+    # Get DVC error message from standard error
+    _, error = p.communicate()
+    error_string = error.decode("utf-8")
+
+    if DVC_INIT_ERROR_MESSAGE in error_string:
+        return False, DVC_INIT_HELP
+    elif DVC_ADD_DOES_NOT_EXIST_MESSAGE in error_string:
+        return False, DVC_ADD_DOES_NOT_EXIST_HELP.format(dvc_add_path=filepath)
+    elif error_string:
+        return False, error_string
+    else:
+        return True, None
+
+
+def _bash_helper(command_list):
+    """
+    Helper for running bash using subprocess and error handling
+    :param command_list: Bash command formatted as a list, no spaces in each element
+    :return: True if no errors, False + error string otherwise
+    """
+    p = subprocess.Popen(command_list, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    _, error = p.communicate()
+    error_string = error.decode("utf-8")
+
+    if error_string:
+        return False, error_string
+
+    return True, None
+
+
+@_app_cli.command("dvc", context_settings=CONTEXT_SETTINGS)
+@click.pass_context
+@click.option(
+    "--init", is_flag=True, required=False, help="Instantiate DVC within a repository"
+)
+@click.option(
+    "--save", is_flag=True, required=False, help="Save built models using dvc"
+)
+@click.option("--checkout", required=False, help="Instantiate DVC within a repository")
```

Comment:
We probably should add a help option, which prints out a similar message to this:
```
Usage: mindmeld [OPTIONS] COMMAND [ARGS]...

  Command line interface for MindMeld.

Options:
  -V, --version        Show the version and exit.
  -v, --verbosity LVL  Either CRITICAL, ERROR, WARNING, INFO or DEBUG
  -h, --help           Show this message and exit.

Commands:
  blueprint  Sets up a blueprint application.
  convert    Converts a Rasa or DialogueFlow project to a MindMeld project
  load-kb    Loads data into a question answerer index.
  num-parse  Starts or stops the local numerical parser service.